.github/workflows/python.yml

```diff
@@ -176,6 +176,9 @@ jobs:
           pip3 install -r python/requirements.txt \
                        -r python/requirements-test.txt \
                        cython
+      - name: Enable Coredump
+        shell: bash
+        run: ulimit -c unlimited -S
```

Comment:
Does this work?
(We need to set `ulimit` in the shell that runs test?)

Reply:
I need to remove it, the PR only focuses on C++ builds.